### DISCUSSION TITLE
Disable Registration

### DISCUSSION THREAD

**Discussion by Curtis2012** [OctoFarm/OctoFarm#758] [Q&A] [answered]
The ability to disable account registration is mentioned in the release notes for v1.1.4, but I have not found any other related documentation. How is this done?

**Comment by NotExpectedYet** [comment, marked as answer]
> 
> 
> The ability to disable account registration is mentioned in the release notes for v1.1.4, but I have not found any other related documentation. How is this done?

In System. 

![image](https://user-images.githubusercontent.com/61137988/136803927-7a2d247c-96d2-4718-a230-c55cea0fcd75.png)
